roles/rpm_version/tasks/main.yml

```diff
@@ -7,6 +7,6 @@
 # This role takes a list of rpms, retrieves the version, and stores it in a
 # dict named g_atomic_host
 #
-- include: version.yml
+- include_tasks: version.yml
```

Comment:
Are you sure we can safely use that here?  I thought since we are looping through the items, we have to use `include_tasks` to be able to get the correct value of `item` each time.

Reply:
I think you're right with include in loops.